src/components/posting/PostingEditor.js

```diff
@@ -190,27 +254,28 @@ const PostingEditor = () => {
                   {item}
                 </div>
               ))}
-            </div>
+            </RegionCheck>
             <WarningMessage>{warnMsgs.regions || ""}</WarningMessage>
-            <h3>[Log option] 날짜별 방문기록 장소를 보여주시겠습니까 ? </h3>
-            <select
+            <LogOptionTitle>
+              [Log option] 날짜별 방문기록 장소를 보여주시겠습니까 ?
+            </LogOptionTitle>
+            <LogOptionItem
               name="logOption"
+              value={logOption}
               onChange={e => setLogOption(e.target.value)}
             >
               <option value="옵션선택">옵션선택</option>
               <option value="true">예</option>
               <option value="false">아니요</option>
-            </select>
+            </LogOptionItem>
             <WarningMessage>{warnMsgs.logOption || ""}</WarningMessage>
```

Comment:
```suggestion
            <WarningMessage>{warnMsgs.logOption || <div />}</WarningMessage>
```
이렇게하면 빈줄이 들어가서 버튼위치가 변하지 않았어요~! 괜찮을까요 ? ?

Reply:
오 좋네요~! 이 부분도 수정하겠습니다~!